training/intel-bootc/Containerfile

```diff
@@ -24,10 +24,10 @@ RUN if [ -f /etc/centos-release ]; then \
        dnf -y update \
        && dnf -y install epel-release \
        && crb enable \
-       && dnf -y install ninja-build pandoc pciutils;\
+       && dnf -y install ninja-build pandoc pciutils tmux;\
     fi \
     && dnf -y update \
-    && dnf install -y dkms ninja-build pandoc pciutils \
+    && dnf install -y dkms ninja-build pandoc pciutils tmux \
```

Comment:
A non blocking comment. This command is executed twice if it is a centos based image.

Reply:
+1 we should just add an else but can come back to it